public/css/style.css

```diff
@@ -0,0 +1,83 @@
+.container {
+  margin: 0 auto;
+  border-width: 14px;
+  border-style: solid;
+  border-color: pink;
+  border-radius: 95px;
+  text-align: center;
+  width: 600px;
+  height: 775px;
+  background-color: lightblue;
+  font-family: 'Comfortaa', cursive;
+  color: black;
+}
+
+.secret-word {
+  font-family: 'Inconsolata', monospace;
+  font-size: 40px;
+  font-weight: bold;
```

Comment:
With most fonts you can install font-weights and then use them here instead of `bold`. So with GoogleFonts, you can click which font weights you need (e.g. 300, 600, 700) and then you would put that number as the value for font-weight.

Reply:
![screen shot 2018-03-06 at 09 34 46](https://user-images.githubusercontent.com/8995723/37024769-b082df70-2121-11e8-822d-b9aff9dbfe58.png)